examples/starter-policy/fastapi/sample_application.py

```diff
@@ -0,0 +1,216 @@
+import oso_sdk
+import os
+from fastapi import Depends, FastAPI, Request
+from fastapi.exceptions import HTTPException
+from oso_sdk.integrations.fastapi import FastApiIntegration
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class ApiKeyNotFound(Exception):
+    """
+    Unable to start application, could not retrieve Oso Cloud API key.
+    """
+
+    pass
+
+
+def api_key() -> str:
```

Comment:
I think observational feedback from users is that this is confusing and we should have them add the key directly in the source code to start. When they start thinking about productionizing, then this is best practice. Other devtool examples also have the API key directly in code.

Reply:
agreed - updated in https://github.com/osohq/oso-python/pull/22/commits/b49ff7836ccc4d9c80f59dfbd622effa128e4996